src/libtrx/game/objects/draw.c

```diff
@@ -191,3 +281,100 @@ void Object_ApplyExtraRotation(
 #undef APPLY_ROTATION
     *extra_rotation = rot_ptr;
 }
+
+void Object_DrawSpriteItem(const ITEM *const item)
+{
+    const RGB_F tint =
+        g_TRVersion == 1 ? Output_GetTint() : (RGB_F) { 1.0f, 1.0f, 1.0f };
```

Comment:
I was thinking of this but maybe we want to colorize underwater sprites in TR2 too? #2093